Replicator/Worker.cc

```diff
@@ -258,7 +258,14 @@ namespace litecore { namespace repl {
         }
     }
 
+    void Worker::setProgressNotificationLevel(int level) {
+        if(_progressNotificationLevel.exchange(level) == level) {
```

Comment:
This caches the new level and returns the old cached level?

Reply:
Yes, that's how all atomic exchanges work as far as I know.